landing-SafeTrust/src/styles/stepper.css

```
@@ -1,312 +1,479 @@
+  padding: 3rem 2rem;
+  box-shadow: 0 12px 36px -10px var(--shadow-color, rgba(15, 23, 42, 0.08));
+  max-width: 1060px;
+  margin: 0 auto;
   position: relative;
+  transition: background-color 0.3s ease, border-color 0.3s ease;
 }
 
-/* Stepper Inner List */
-.stepper-list {
+html.dark .stepper-card,
+:global(html.dark) .stepper-card {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)